tests/Unit/OutputFormatTest.php

```diff
@@ -35,7 +35,7 @@ public function getStringQuotingTypeInitiallyReturnsDoubleQuote(): void
      */
     public function setStringQuotingTypeSetsStringQuotingType(): void
     {
-        $value = 'x';
+        $value = '"';
```

Comment:
Isn't this now the same as the default value?

Reply:
It is indeed! Changed to a single quote now.